client/branded/src/components/panel/TabbedPanelContent.tsx

```diff
@@ -284,7 +291,7 @@ export const TabbedPanelContent = React.memo<TabbedPanelContentProps>(props => {
         )
     }, [items, hash, currentTabID])
 
-    if (!areExtensionsReady) {
+    if (!areExtensionsReady && extensionsController !== null) {
```

Comment:
```suggestion
    if (!areExtensionsReady) {
```

It seems that if `extensionsController` is `null`, `areExtensionsReady` will equal to `true` according to its definition on line 144. Correct?

Reply:
👍  I need to check if we still need the test to make TypeScript happy though!